recipe/meta.yaml

```diff
@@ -65,6 +65,7 @@ requirements:
     - xorg-libxrandr       # [linux]
     - xorg-libxrender      # [linux]
     - xorg-libxtst         # [linux]
+    - xorg-libxxf86vm      # [linux]
```

Comment:
Do we know why this is needed?

Reply:
Looks like it is not. Thanks for the catch.